### ISSUE TITLE
Rework Script, Fix Bugs, Add Multilanguage Support & Update Version System

### ISSUE THREAD

**Issue by LetsTiger** [TGScripts/tg_discordwebhook#1]
- Rework & Improve the Script (Main Functions & Additions)
- Fix Bugs
- Add Multilanguage Support (English Translation)
- Update Version System to "M.M.P"

**Comment by LetsTiger** [comment]
> * Rework & Improve the Script (Main Functions & Additions)
> * Fix Bugs
> * Add Multilanguage Support (English Translation)
> * Update Version System to "M.M.P"

Reviewed the script and found no bugs, script version system will be updated to "M.M.P" and a disclaimer that this only works when called from the server will be added to the README.md.

**Comment by LetsTiger** [comment]
Correction of https://github.com/TGScripts/tg_discordwebhook/issues/1#issuecomment-2525675687: *"Reviewed the script and found no bugs"*
-> Fixed Typo in `fxmanifest.lua`: `SentTextToDiscord -> SendTextToDiscord`


Before:
```lua
exports {
	'SendEmbedToDiscord',
	'SentTextToDiscord',
	'SendEmbedAndTextToDiscord',
}
```
After:
```lua
exports {
	'SendEmbedToDiscord',
	'SendTextToDiscord',
	'SendEmbedAndTextToDiscord',
}
```

Resolved in: [v1.2.1](https://github.com/TGScripts/tg_discordwebhook/releases/tag/v1.2.1)
